include/boxfort.h

```diff
@@ -288,7 +294,7 @@ typedef const struct bxf_start_params_s *bxf_start_params;
     (Sandbox),                            \
     &(struct bxf_start_params_s) { .bxfi_sentinel_ = 0, __VA_ARGS__ }))
 BXF_API int bxf_start_struct(bxf_instance **instance,
-        bxf_sandbox *sandbox, bxf_start_params params);
+        struct bxf_sandbox_s *sandbox, bxf_start_params params);
```

Comment:
It's somewhat weird that the sandbox would be mutated by a call to start. In principle, one could reuse the sandbox object for multiple calls to start.

I'd rather add two new fields (suspend and debugger) to the private `bxfi_sandbox` type which would be initialized by `sandbox->suspend` & `sandbox->debugger`, and would be used in place of the aforementioned fields.

Reply:
Good idea, thanks.

I've seen that the struct is copied in most cases, but changing the signature of`bxf_start_struct()` was really risky, I agree.
Sorry. I'm fixing it.

Reply:
Done. I ended up using a single private flag in `bxfi_sandbox`.